alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/notice-of-intent-details.component.html

```diff
@@ -150,11 +156,21 @@ <h3>Proposal</h3>
 </section>
 <section *ngIf="showFullApp">
   <h3>Additional Proposal Information</h3>
-  <app-roso-additional-information
+  <app-additional-information
```

Comment:
Since all of these are the same, can you have a single one with no ngIf?

Reply:
yes, you right, too many copy paste 🤦